### PR TITLE
Return if text doesn't contain '<'

### PR DESCRIPTION
**Pull request by ivandrofly**
`text.IndexOfAny(tags, StringComparison.Ordinal);` is little bit expensive

### CODE REVIEW

File: src/Core/HtmlUtil.cs [hunk 1]
```diff
@@ -402,7 +402,7 @@ public static bool StartsWithUrl(string text)
 
         internal static string FixUpperTags(string text)
         {
-            if (string.IsNullOrEmpty(text))
+            if (string.IsNullOrEmpty(text) && !text.Contains('<'))
```
Comment (niksedk) [inline src/Core/HtmlUtil.cs]: should that have been "|| !text.Contains('<')" ?

**Comment by ivandrofly** [comment]
Fixed /cc @niksedk

**Comment by ivandrofly** [comment]
Safe /cc @niksedk

**Comment by niksedk** [comment]
I just could not see any performance difference - even at 10.000.000 tries...
Moving the string array out actually gave a larger improvement - even if the input string does not contain '<':
private static readonly string[] Tags = { "<I>", "<U>", "<B>", "<FONT", "</I>", "</U>", "</B>", "</FONT>" };

**Comment by ivandrofly** [comment]
Okay, I thought it wouldn't bee necessary to waste time checking for all the tags

**Comment by niksedk** [comment]
It's no waste of time... but don't optimize without reason :)

**Comment by ivandrofly** [comment]
:+1:

**Pull request closed without merging**